kyo-core/shared/src/main/scala/kyo/randoms.scala

```diff
@@ -34,8 +115,8 @@ object Randoms:
     val nextInt: Int < IOs =
         local.use(_.nextInt)
 
-    def nextInt[S](n: Int < S): Int < (S & IOs) =
```

Comment:
I've been moving away from providing effectful parameters where not necessary. It introduces overhead and I think being more explicit about where binds happen helps usability.